src/index.ts

```diff
@@ -413,13 +396,15 @@ export class GalleryComponent extends BaseComponent {
       thumb.initialHeight = medianHeight;
     }
 
-    this._$thumbs.link($.templates.galleryThumbsTemplate, this._thumbs);

```

Comment:
this is previously the line that was causing us the issue where the jsrender templating engine would take the template and build out the html from it.